_includes/help/nav_sidenav.html

```diff
@@ -0,0 +1,17 @@
+<nav class="sm-col-right sm-col-4 sm-show">
+  <ul class="list-reset pt2 red nav">
```

Comment:
Same question about `red` and `nav` here - not 100% if those are being used any longer.